_posts/hdubbe/2021-11-24-project-update

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Could you add a `.md` file extension to the name of this file? Otherwise, any markdown you use in your file won't display properly.

Reply:
Hi Misha, Could you remind me how to edit a commit again? I can't seem to get to the right page :/ 
Thanks!

Reply:
Certainly! First, go to the "files changed" on this pull request.

<img width="517" alt="image" src="https://user-images.githubusercontent.com/5872061/144283450-fe6c1138-e5e8-4f39-a3e1-8dd47cc0a87b.png">

Then, select the "Edit file" option.

![image](https://user-images.githubusercontent.com/5872061/144283307-bb99cbdf-7ba7-4f80-9a0f-4bb00b4234ed.png)

From there you can edit the contents of the file or its name by clicking in the box.